chromium/rules.js

```diff
@@ -252,7 +250,7 @@
       var platform = ruletag["platform"]
       if (platform) {
         default_state = false;
-        if (platform == "mixedcontent" && background.enableMixedRulesets.enable) {
+        if (platform == "mixedcontent" && enableMixedRulesets.enable) {
```

Comment:
Btw why do you need this indirection and not just `enableMixedRulesets` being a boolean directly?

Reply:
This won't work since it needs to be the same object.

Reply:
This can be done with `get()` and `set()` methods instead.

Reply:
Or you can just use `exports.enableMixedRulesets`. Then it will be same inside and outside.